src/component/plasmic/flip_card/PlasmicUpdate.tsx

```diff
@@ -27,8 +27,8 @@ import {
   StrictProps,
   deriveRenderOpts
 } from "@plasmicapp/react-web";
-import Metric from "../../Metric"; // plasmic-import: _i6uD1XPzdbux6R/component
-import IconButton from "../../IconButton"; // plasmic-import: odPjbfT2kyJgB_S/component
+import Metric from "../../metric/index"; // plasmic-import: _i6uD1XPzdbux6R/component
+import IconButton from "../../iconButton/index"; // plasmic-import: odPjbfT2kyJgB_S/component
```

Comment:
Would be cool to get this cleaner as well. 😬 What is actually with the funny comments? Are they essential? 

```suggestion
import metric from "component/metric"; // plasmic-import: _i6uD1XPzdbux6R/component
import iconbutton from "component/iconbutton"; // plasmic-import: odPjbfT2kyJgB_S/component
```